src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -37,6 +37,8 @@
 
 logger = get_logger(__name__)
 
+FILE = ""
+
 def acr_build_show_logs(cmd,
```

Comment:
@djyou / @northtyphoon / @NatalieGong what's the context of this PR? The size limit seems like something better handled off on the server side to me so we can unblock people if needed. Images can easily reach 800MB, even on Linux builds - probably want the freedom to support multiple GBs.

Reply:
The file is written to storage directly. I suggest we have a reasonable limit. Upload 500MB tar file to build is the very rare case. If it's beyond, I guess the user likely include a lot of intermediate build output files.

Reply:
After thinking a bit more on this, limit is something which we will have to remove. We did this for the registry, so would avoid this all together until we know better.